src/SFA.DAS.EmployerIncentives.Api/Controllers/ApplicationCommandController.cs

```diff
@@ -21,5 +23,20 @@ public async Task<IActionResult> CreateIncentiveApplication([FromBody]CreateInce
             await SendCommandAsync(new CreateIncentiveApplicationCommand(request.IncentiveApplicationId, request.AccountId, request.AccountLegalEntityId, request.Apprenticeships));
             return Created($"/applications/{request.IncentiveApplicationId}", null);
         }
+
+        [HttpPost("/submit-application")]
```

Comment:
That isn't a restful url...don't ask me what the correct url would be tho! 😉 
Maybe something like patch /applications/{id}/